src/UKHO.ERPFacade.API/Helpers/EncContentSapMessageBuilder.cs

```diff
@@ -3,15 +3,17 @@
 using UKHO.ERPFacade.Common.IO;
 using UKHO.ERPFacade.Common.Logging;
 using UKHO.ERPFacade.Common.Models;
+using UKHO.ERPFacade.Common.Providers;
 
 namespace UKHO.ERPFacade.API.Helpers
 {
     public class EncContentSapMessageBuilder : IEncContentSapMessageBuilder
     {
-        private readonly ILogger<EncContentSapMessageBuilder> _logger;
+        private static ILogger<EncContentSapMessageBuilder> s_logger;
```

Comment:
Can we keep this object variable as a readonly?

Reply:
Resolved